custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -436,7 +437,7 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         vm.selectedLevel = 1;
         vm.selectedMonth = new Date().getMonth() + 1;
         vm.selectedYear = new Date().getFullYear();
-        vm.selectedIndicator = 1;
+        vm.selectedIndicator = vm.selectedIndicator;
```

Comment:
I think in this place we should have 1, because that function is executed when we click reset form button which reset all fields. I think you should consider an option to add parameter in this function to recognize from where this function was called and based on this set ```vm.selectedIndicator``` to 1 or current selection.